easybuild/easyconfigs/e/EMAN2/EMAN2-2.21a-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'EMAN2'
+version = '2.21a'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
+ with a primary focus on processing data from transmission electron microscopes. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/cryoem/eman2/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['EMAN2-2.21a.patch']
+checksums = [
+    '8a7310b60852220103a6676c7116f4119b290776de572a1510b982783bff46f7',  # v2.21a.tar.gz
+    '7399b4645c7fda87dc102f4ce3f7011764e40833984bb6cac025f840ed177a19',  # EMAN2-2.21a.patch
+]
+
+builddependencies = [('CMake', '3.10.2')]
+
+dependencies = [
+    ('Boost', '1.63.0', versionsuffix),
+    ('freetype', '2.9'),
+    ('FTGL', '2.1.3-rc5'),
```

Comment:
Maybe add a comment to clarify why we're using a release candidate here? Looks a bit fishy at first sight.

Reply:
@boegel it's the latest version since 2008, and is also the version included in ubuntu 16.04 and centos 7.4.